README.md

```diff
@@ -5,7 +5,7 @@
 > Please see our [iOS 13 Migration Guide](https://github.com/twilio/twilio-voice-ios/blob/Releases/iOS-13-Migration-Guide.md) for the latest information on iOS 13.
 
 ## Get started with Voice on iOS:
-* [Quickstart](#quickstart) - Run the quickstart app
+* [Quickstart](#quickstart) - Run the swift quickstart app

```

Comment:
In the Video repo, we left the header up here alone and down in the section that it links to, we added subsections for both the Swift app and one for the Obj-C app.